motor/providers/aws/provider.go

```diff
@@ -94,14 +94,18 @@ func New(pCfg *providers.Config, opts ...ProviderOption) (*Provider, error) {
 	// gather information about the aws account
 	identity, err := CheckIam(t.config)
 	if err != nil {
+		log.Warn().Err(err).Msg("could not gather details of AWS account")
```

Comment:
i think we're going to double warn with this?  
basically the logic here is 
no region was provided -> try us-east-1 -> that doesn't work, then try gov region. (a region is required by the api) invalid credentials won't work in either case

Reply:
we could probably just remove both warns if we're returning the err

Reply:
yeah, i wasn't sure how best to handle the two errors. it would definitely show two log messages, but i figured there was the remote possibility that the error for us-east-1 was different than govcloud (maybe some DNS or something for one but an actual credentials error for the other...).

what do you think about burying the messages behind debug, and returning the "last" error encountered?

Reply:
👍  that sounds good!